From dc498fec8e77bda0397e8893cd603f1876c070ce Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:03:57 -0400
Subject: [net] usb/cdc_ncm: remove unused udev field

Message-id: <1410887463-20674-166-git-send-email-dzickus@redhat.com>
Patchwork-id: 95132
O-Subject: [RHEL7 PATCH 165/591] net: cdc_ncm: remove unused udev field
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit de5bee2720776989060b9686e6a89e938a346345
Author: Bjorn Mork <bjorn@mork.no>
Date:   Fri Nov 1 11:16:43 2013 +0100

    net: cdc_ncm: remove unused udev field

    We already use the usbnet udev field everywhere this could have
    been used.

    Cc: Alexey Orishko <alexey.orishko@gmail.com>
    Signed-off-by: Bjorn Mork <bjorn@mork.no>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/usb/cdc_ncm.c b/drivers/net/usb/cdc_ncm.c
index e39e767..9cdd762 100644
--- a/drivers/net/usb/cdc_ncm.c
+++ b/drivers/net/usb/cdc_ncm.c
@@ -388,8 +388,6 @@ int cdc_ncm_bind_common(struct usbnet *dev, struct usb_interface *intf, u8 data_
  buf = intf->cur_altsetting->extra;
  len = intf->cur_altsetting->extralen;
 
- ctx->udev = dev->udev;
-
  /* parse through descriptors associated with control interface */
  while ((len > 0) && (buf[0] > 2) && (buf[0] <= len)) {
 
diff --git a/include/linux/usb/cdc_ncm.h b/include/linux/usb/cdc_ncm.h
index 5c47bd9..059dcc9 100644
--- a/include/linux/usb/cdc_ncm.h
+++ b/include/linux/usb/cdc_ncm.h
@@ -98,7 +98,6 @@ struct cdc_ncm_ctx {
  const struct usb_cdc_union_desc *union_desc;
  const struct usb_cdc_ether_desc *ether_desc;
 
- struct usb_device *udev;
  struct usb_interface *control;
  struct usb_interface *data;
 
-- 
1.7.1